cmd/http/headers.go

```diff
@@ -66,6 +66,12 @@ const (
 	AmzTagCount      = "x-amz-tagging-count"
 	AmzTagDirective  = "X-Amz-Tagging-Directive"
 
+	// S3 transition restore
+	AmzRestore            = "X-Amz-Restore"
+	AmzRestoreExpiryDays  = "X-Amz-Restore-Expiry-Days"
+	AmzRestoreRequestDate = "X-Amz-Restore-Request-Date"
+	AmzRestoreOutputPath  = "X-Amz-Restore-Output-Path"
```

Comment:
have you verified if this lower cased in case of AWS s3?

Reply:
is there a difference? I see the AWS docs showing lowercase but the output of `aws s3api restore-object` returned camel cased which is what I based the header names on.

Reply:
We are basically trying to send exactly how AWS might send, i.e on the wire. just to be sure to avoid confusion with poorly implemented SDKs. Ideally, HTTP headers are case insensitive it is to save a conversation.